cmd/natc/natc.go

```diff
@@ -192,6 +207,29 @@ func (c *connector) serveDNS() {
 	}
 }
 
+func lookupDestinationIP(req *dnsmessage.Message) ([]netip.Addr, error) {
+	if len(req.Questions) == 0 {
+		return nil, nil
+	}
+	q := req.Questions[0]
+	switch q.Type {
+	case dnsmessage.TypeAAAA, dnsmessage.TypeA:
+		netIPs, err := net.LookupIP(q.Name.String())
```

Comment:
do you need to care about AAAA and A separately here? not sure if there is knob for `net.Resolver` that you could use.

Reply:
net.LookupIP returns a slice of ipv6 and ipv4 addresses combined when I try it.